care/emr/reports/context_builder/builders/file_upload.py

```
@@ -0,0 +1,88 @@
+from care.emr.models import Encounter
+from care.emr.models.file_upload import FileUpload
+from care.emr.reports.context_builder.base import Field, QuerysetContextBuilder
+from care.emr.reports.context_builder.registry import context_builder_registry
+from care.emr.reports.context_builder.utils import format_datetime
+
+
+class FileUploadContextBuilder(QuerysetContextBuilder):
+    model = FileUpload
+    depends_on = ["encounter_id"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)